backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/UsersController.java

```diff
@@ -25,26 +34,71 @@
 @Tag(name = "Users", description = "CRUD endpoint for users.")
 public class UsersController {
 
+	private static final Logger log = LoggerFactory.getLogger(UsersController.class);
+
 	private final SubscriptionModelAssembler subscriptionModelAssembler;
 
 	private final SubscriptionService subscriptionService;
 
-	public UsersController(SubscriptionModelAssembler subscriptionModelAssembler, SubscriptionService subscriptionService) {
+	private final SubscriptionModelMapper mapper;
+
+	public UsersController(SubscriptionModelAssembler subscriptionModelAssembler, SubscriptionService subscriptionService, SubscriptionModelMapper mapper) {
 		Assert.notNull(subscriptionModelAssembler, "subscriptionModelAssembler is required; it must not be null");
 		Assert.notNull(subscriptionService, "subscriptionService is required; it must not be null");
 		this.subscriptionModelAssembler = subscriptionModelAssembler;
 		this.subscriptionService = subscriptionService;
+		this.mapper = mapper;
```

Comment:
A bit of a nitpick (since it shouldn't happen during normal runtime) but for correctness we should have a `.notNull()` assertion for the mapper.

Reply:
added